rawsend.c

```diff
@@ -58,6 +58,11 @@ static unsigned ip_header_checksum (const void * header);
 static uint16_t udp_sum_calc (uint16_t, uint32_t, uint16_t, uint32_t, uint16_t, const void *);
 static uint16_t udp_sum_calc_v6 (struct in6_addr, uint16_t, struct in6_addr, uint16_t, uint16_t, const char *);
 
+struct in6_pktinfo {
```

Comment:
You don't need to define this here ( and you likely shouldn't. To get this to work you need to compile with `-D _GNU_SOURCE`

Reply:
https://sourceware.org/git?p=glibc.git;a=blob;f=inet/netinet/in.h;h=1633bc64e43b22bb6600341096be0e4b3c6218f2;hb=HEAD#l539